tests/functional/measurements_concat.t

```diff
@@ -0,0 +1,42 @@
+Integration tests for augur measurements export.
+
+  $ pushd "$TESTDIR" > /dev/null
+  $ export AUGUR="../../bin/augur"
+
+Measurements concat for two measurements JSONs, each with a single collection.
+
+  $ ${AUGUR} measurements concat \
+  >   --jsons measurements_concat/single_collection_measurements_1.json measurements_concat/single_collection_measurements_2.json \
+  >   --default-collection collection_1 \
+  >   --output-json "$TMP/two_collections_measurements.json" &>/dev/null
+
+  $ python3 "$TESTDIR/../../scripts/diff_jsons.py" measurements_concat/two_collections_measurements.json "$TMP/two_collections_measurements.json"
+  {}
+
+Measurements concat for two measurements JSONs, where one has multiple collections.
+
+  $ ${AUGUR} measurements concat \
+  >   --jsons measurements_concat/two_collections_measurements.json measurements_concat/single_collection_measurements_3.json \
+  >   --default-collection collection_1 \
+  >   --output-json "$TMP/multiple_collections_measurements.json" &>/dev/null
+
+  $ python3 "$TESTDIR/../../scripts/diff_jsons.py" measurements_concat/multiple_collections_measurements.json "$TMP/multiple_collections_measurements.json"
+  {}
+
+Measurements concat for measurements JSONs that have collections that share the same key.
+This is expected to fail.
+
+  $ ${AUGUR} measurements concat \
+  >   --jsons measurements_concat/single_collection_measurements_1.json measurements_concat/single_collection_measurements_1.json \
+  >   --default-collection collection_1 \
+  >   --output-json "$TMP/multiple_collections_measurements.json" &>/dev/null
+  [1]
```

Comment:
> I would only write stdout to /dev/null and let stderr get consumed by Cram

Ah, I did not think to do this 🤦‍♀️ . I just redirected everything because I didn't want to include the "Validating..." messages. Addressed in https://github.com/nextstrain/augur/pull/879/commits/bd805bad1a9dd1e9ff29cab826dba2fff306d0dc